1-mandatory/1-writers.js

```diff
@@ -62,7 +65,11 @@ Exercise 2:
 */
 
```

Comment:
Nice 👍 
Can you think of how to do this using multiple array methods? (maybe `filter` first, and then `forEach`)